man/systemd-gpt-auto-generator.xml

```diff
@@ -196,6 +196,40 @@
     <citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry>.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Kernel Command Line</title>
+
+    <para><filename>systemd-gpt-auto-generator</filename> understands the following kernel command line
+    parameters:</para>
+
+    <variablelist class='kernel-commandline-options'>
+
+      <varlistentry>
+        <term><varname>systemd.gpt_auto</varname></term>
+        <term><varname>rd.systemd.gpt_auto</varname></term>
+
+        <listitem><para>Those options take an optional boolean argument, and default to yes.
+        The generator is enabled by default, and a negative value may be used to disable it.
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>root=</varname></term>
+
+        <listitem><para>When used with the special value <literal>gpt-auto</literal>, automatic discovery of
+        the root parition based on the GPT partition type is enabled.  Any other value disables this
+        generator.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>rw</varname></term>
+        <term><varname>ro</varname></term>
+
+        <listitem><para>Mount the root partition read-write or read-only.</para></listitem>
```

Comment:
this needs clarification, as this does not have an effect on the final state of the root rw state, since /etc/fstab will override it, due to historical reasons... i.e. this should say "Note that unlike most kernel command line options these settings do not override configuration in the file system." or something like that